src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  gap: 48px 46px;
+  grid-template-columns: repeat(var(--column-count), 1fr);
+  justify-items: center;
+
+  @include screen-xs {
+    --column-count: 2;
+  }
+
+  @include screen-sm {
```

Comment:
The mixin `screen-sm` is used here. Make sure this mixin is defined and correctly imported in your SCSS setup to handle media queries for small screens.